woocommerce/woocommerce
# Spacing adjustments for coupon code in the Cart and Checkout blocks

### Changes proposed in this Pull Request:





This PR introduces the following changes:

- Add a top margin of 4px to the coupon code `.wc-block-components-totals-discount__coupon-list`.
- Ensure that the top margin is consistent on both the cart and checkout pages.

Closes #49480.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `/wp-admin/edit.php?post_type=shop_coupon` and create a test coupon.
2. Go to the frontend.
3. Add a product to the cart.
4. Go to the cart page with the Cart block.
5. Provide the test coupon from step 1.
6. Verify that the coupon chip has a top margin of 4px.
7. Go to the checkout page with the Checkout block.
8. Verify that the coupon chip also has a top margin of 4px.

<table>
<tr>
<td valign="top">Cart block / Before:
<br><br>
<img width="357" alt="Screenshot 2024-07-16 at 15 48 14" src="https://github.com/user-attachments/assets/e2ff426c-7a5c-46a4-bfbf-bb7d6d1fb3f2">
</td>
<td valign="top">Cart block / After:
<br><br>
<img width="366" alt="Screenshot 2024-07-16 at 15 40 59" src="https://github.com/user-attachments/assets/9303c56c-9f00-4ecd-b245-f897548175a8">
</td>
</tr>
</table>

<table>
<tr>
<td valign="top">Checkout block / Before:
<br><br>
<img width="350" alt="Screenshot 2024-07-16 at 15 47 59" src="https://github.com/user-attachments/assets/fb717721-f5fb-4c9b-8e35-81452dcc4f65">
</td>
<td valign="top">Checkout block / After:
<br><br>
<img width="353" alt="Screenshot 2024-07-16 at 15 41 14" src="https://github.com/user-attachments/assets/a01a8d76-99f6-4317-8774-ea3af0ea7cbc">
</td>
</tr>
</table>



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Adjust top margin of the coupon code in the Cart and Checkout blocks.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
